### DISCUSSION TITLE
Does altitude matter for LatLonAltToXY?

### DISCUSSION THREAD

**Discussion by terra-incognita** [SimDynamX/SpaceTeamsPRO#82] [2024 Lunar SAR Competition Q&A] [answered]
I'm trying to send a vehicle to a particular spot by using `ET.LatLonAltToXY()` on a set of lat/lon coordinates from Moon Trek, but it doesn't seem to go where I think it should go. I'm just passing in 0 as the altitude.  Do you need a specific altitude for this to work?

**Comment by connorjak** [comment]
How far off is it? There may be some small error in the projection math from a mismatch in assumed altitude, but I think it would be only up to a few meters. You can check where the lat/lon/alt readout in the sim UI sees the position and compare it to the values that were passed in to `LatLonAltToXY()`

**Reply by terra-incognita** [reply]
It seems pretty far off:
```python
lla_crater = st.PlanetUtils.LatLonAlt(-89.36777,-129.93995, 0)
xy_crater = ET.LatLonAltToXY(lla_crater)
move_tl = TG.Task("MoveTL", Command_MoveToCoord(LTV1, xy_crater, "MoveTL"))
LTV1_task_graph.add_task(move_tl, [])
```

But when I get right on top of LTV1 when it completes the move it ends up in:
![image](https://github.com/user-attachments/assets/63f8db42-f6ea-416d-a97e-fed78528b8d2)

**Comment by connorjak** [comment, marked as answer]
That is odd; I got similar behavior!

![image](https://github.com/user-attachments/assets/89f77d2d-cbed-4fc9-91a5-09f35d7c53ac)

One thing is those coords are far outside of the search area (where obstacles show up). There may be a math error in the Coord / LatLonAlt math internally that doesn't show up when just operating around the search area.

Seems like the LTV runs out of battery on the coords you see. The actual xy output of that code is `-109487.55132688943, 275221.5652213214` (in meters). This is a significant portion of the moon's radius

**Reply by connorjak** [reply]
so in summary just try to keep the lat/lon coords near the search area
